integration/env/common.go

```diff
@@ -0,0 +1,32 @@
+package env
+
+import (
+	"os"
+)
+
+const (
+	// EnvVarCircleCI is the process environment variable representing the
+	// CIRCLECI env var.
+	EnvVarCircleCI = "CIRCLECI"
+	// EnvVarKeepResources is the process environment variable representing the
+	// KEEP_RESOURCES env var.
+	EnvVarKeepResources = "KEEP_RESOURCES"
```

Comment:
There are some others like `CIRCLE_SHA1`, that I want to get done - but this works as a start.

Reply:
I didn't add `CIRCLE_SHA1` because it isn't used yet. But I've added because we will need it. Also I think it makes sense to standardise this code.